flytekit/annotated/reference_task.py

```diff
@@ -36,10 +36,10 @@ def get_task_structure(self) -> SdkTask:
 
 def reference_task(
```

Comment:
Ugh, I should change this...  i'll add it to the list.  these shouldn't be optional.